src/encoded/audit/experiment.py

```diff
@@ -1904,6 +1904,24 @@ def audit_experiment_gtex_biosample(value, system):
     return
 
 
+@audit_checker('Experiment', frame=['object'])
```

Comment:
Idan - I agree this does what you ask it to... but there is no guarantee that having a GEO dbxref means it was actually submitted...   I guess it's close enough